packages/anchors/demo/anchors.stories.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import '../dist/index.css';
+import { Grid } from '@zendeskgarden/react-grid';
+import React from 'react';
+import { useTheme } from 'styled-components';
+
+export default {
+  title: 'Components/Anchors'
+};
+
+const toClassName = (theme, base, style = []) => {
+  let retVal = theme.colors.base === 'dark' ? `${base} c-anchor--dark` : base;
+
+  if (style.length > 0) {
+    retVal += ` ${style.join(' ')}`;
+  }
+
+  return retVal;
+};
+
+const toClassSelector = (theme, base, style = []) => {
+  let retVal = theme.colors.base === 'dark' ? `${base}.c-anchor--dark` : base;
+
+  if (style.length > 0) {
+    retVal += `.${style.join('.')}`;
+  }
+
+  return retVal;
+};
```

Comment:
In the future, would it make sense to move these utility functions into `./.storybook/utils` to keep things DRY?